Add using EXPLAIN ANALYZE and Grafana to coprocessor cache (#4086)
pingcap/docs: coprocessor-cache.md

## Check the cache effect

You can check the cache effect of Coprocessor by executing `EXPLAIN ANALYZE` or viewing the Grafana monitoring panel.

### Use `EXPLAIN ANALYZE`

> Users can view the cache hit rate in [Operators for accessing tables](/choose-index.md#operators-for-accessing-tables) through [statement `EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md). See the following example:

Reviewer: To keep the person style consistent in this document. No need to strictly follow the Chinese version.
```suggestion
You can view the cache hit rate in [Operators for accessing tables](/choose-index.md#operators-for-accessing-tables) by using the [`EXPLAIN ANALYZE` statement](/sql-statements/sql-statement-explain-analyze.md). See the following example:
```

Author: OK, thanks:)

| └─TableRowIDScan_5(Probe)     | 262400.00 | 0       | cop[tikv] | table:t                | time:0ns, loops:0                                                                                                                                                                                                                                        | keep order:false, stats:pseudo | N/A                   | N/A  |
+-------------------------------+-----------+---------+-----------+------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------+-----------------------+------+
3 rows in set (0.62 sec)
```

The column `execution info` of the execution result gives the `copr_cache_hit_ratio` information, which indicates the hit rate of the Coprocessor Cache. The `0.75` in the above example means that the hit rate is about 75%.

### View the Grafana monitoring panel

> In Grafana monitoring, the **copr-cache** panel can be seen in the `distsql` subsystem under the `tidb` namespace. This panel monitors the number of hits, misses, and cache discards of the Coprocessor Cache in the entire cluster.

Reviewer: Generally, use active voice instead of passive voice where applicable or appropriate.
```suggestion
In Grafana, you can see the **copr-cache** panel in the `distsql` subsystem under the `tidb` namespace. This panel monitors the number of hits, misses, and cache discards of the Coprocessor Cache in the entire cluster.
```
